docs/deployment-manager-guide.md

```diff
@@ -1,65 +1,21 @@
-## Generating a Deployment Manager template
+## Generating Deployment Manager templates using the `mpdev` tool
 
-The `mpdev` tool can autogenerate Deployment Manager templates from an
-[autogen specification](./autogen-reference.md). Follow these steps to create a
-Deployment Manager template.
+You can use the `mpdev` tool to create Deployment Manager template files, also known as a _deployment package_, for VM
```

Comment:
> also known as a _deployment package_,

the pluralization here should match that of the previous clause ("template files")